README.rst

```diff
@@ -8,7 +8,9 @@
 • `Tutorial <https://dvc.org/doc/get-started>`_
 • `Mailing List <https://sweedom.us10.list-manage.com/subscribe/post?u=a08bf93caae4063c4e6a351f6&id=24c0ecc49a>`_
 
-|Release| |CI| |Maintainability| |Coverage| |Donate| |Conda| |Snap| |DOI|
+|Release| |CI| |Maintainability| |Coverage| |Donate| |DOI|
+
+|Snap| |Choco| |Brew| |Conda| |PyPI| |Packages|
```

Comment:
Do we really need this number of badges? Maybe, we could just put a same page link to `Installation` section?

Reply:
@skshetry Good point. So far the idea was that it is conveinient to see all of those badges, as you are intuitively looking for them whenever you are checking out a new project. Hiding them will make it less visible, so I'm not sure if it is a good thing. It does get a bit crowded, you are right.

Reply:
Indeed, though if we want to drop things arguably `|Release| |CI| |Maintainability| |Coverage|` are least relevant to end users.

Reply:
Ok, looking at readme now, I do see that it is way too crowded with installation badges :(

> Indeed, though if we want to drop things arguably |Release| |CI| |Maintainability| |Coverage| are least relevant to end users.

That is not true, maintainability/coverage/ci all matters, as it is a sign of a well-maintained project. Those should not be removed.

Reply:
Doesn't look too crowded to me:

![image](https://user-images.githubusercontent.com/10780059/75574032-e772c580-5a55-11ea-80fd-6b4ed8fdac49.png)

Showing we support different installation options means more users likely to bother reading on an actually install. Maintainability/coverage/ci (in terms of users, not devs) are either completely ignored or assumed to be perfect - in both cases they don't necessarily need to be at the top of the readme if we want to drop some badges.

Reply:
I feel that 19 links at the top is too many. It makes it hard to scan through when the objective of having top-links and badges is to be quick and easy.